content/_data/roadmap/roadmap.yml

```
@@ -118,6 +118,25 @@ categories:
       description: Currently Jenkins has no documented Windows Support policy. We want to add one and to deprecate/remove support for old platforms
       status: near-term
       link: https://issues.jenkins-ci.org/browse/JENKINS-61865
+  - name: Pipeline Authoring
+    link: /sigs/pipeline-authoring
+    initiatives:
+    - name: Pipeline development in IDE
+      status: future
+      link: https://issues.jenkins-ci.org/browse/JENKINS-35396
+    - name: Syntax Improvements
+      status: future
+      link: https://issues.jenkins-ci.org/browse/JENKINS-55287?filter=-3
+    - name: Static Code Analysis and Linting
+      status: future
+      link: https://issues.jenkins-ci.org/browse/JENKINS-52939?filter=-3
```

Is there special significance to `filter=-3`?
```suggestion
      link: https://issues.jenkins-ci.org/browse/JENKINS-52939
```